components/menu/menu.js

```diff
@@ -27,6 +27,7 @@ class Menu extends HierarchicalViewMixin(LitElement) {
 			:host {
 				box-sizing: border-box;
 				display: block;
+				vertical-align: top;
```

Comment:
This is the same v-align fix we used to stop it from happening in the initial load. The fix works - but `lint` hates it, because this property doesn't do anything with `display: block`. The issue is that after opening and closing the dropdown once, when you reopen it, [this is applied](https://github.com/BrightspaceUI/core/blob/master/components/hierarchical-view/hierarchical-view-mixin.js#L38) from the hierarchical view- so it's displayed as an inline-block from that point. (and the mystery padding sets in).

Reply:
Do we know why we set it to `inline-block` and not `block` in hierarchical-view, or why we use `block` here and not `inline-block`?  I remember investigating this with Margaree and we both thought it was odd to have it be `block` the first time it's opened, and `inline-block` the second time... @dbatiste do you remember if this was intentional?

Reply:
Could you move it into that CSS rule then so the linter doesn't complain? Like:

```
:host([shown]) {
	display: inline-block;
	vertical-align: top;
}
```

Reply:
If we don't anticipate any issues from that rule being applied elsewhere, for sure.

Reply:
Let's put it there -- that way it's next to the line that's causing the problem, and the linting will pass.

Reply:
You could even put a little comment:

```
vertical-align: top; /* DE37329: required to prevent extra spacing caused by inline-block */
```

If this is easily reproducible, it would also be great to add a visual diff test for it so that we don't accidentally regress in the future.